src/main/java/org/fundacionjala/coding/ariel/Kata1.java

```diff
@@ -0,0 +1,22 @@
+package org.fundacionjala.coding.ariel;
+
+/**
+ * Kata Ariel Gonzales.
+ */
+public class Kata1 {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)